### PR TITLE
feat(sample_sensor_kit_launch): pass container to velodyne nodes

### PR DESCRIPTION
**Pull request by kaancolak**
Signed-off-by: Kaan Colak <kcolak@leodrive.ai>

## Description

 Working with an individual container for each lidar increases network traffic and CPU utilization. Make it possible to pass `pointcloud_container` to Velodyne nodes.

 When working with logging_simulator.launch.xml, it decreased network utilization around ~30-40 MB/s . 

 When setting `launch_driver:=true` on `logging_simulator.launch.xml` or `autoware.launch.xml` it looks working without problem, but needs to test with real device. I will test it to make sure multiple velodyne drivers working in the same container with different namespaces without any problems.

Relevant issues and PR :
[https://github.com/autowarefoundation/autoware.universe/issues/1228](https://github.com/autowarefoundation/autoware.universe/issues/1228 )
[https://github.com/autowarefoundation/autoware.universe/pull/2114](https://github.com/autowarefoundation/autoware.universe/pull/2114)

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by kaancolak** [comment]
Hello @mitsudome-r Mitsudome-san, 

Relevant PR was merged today: https://github.com/autowarefoundation/autoware.universe/pull/2114

Could you review this PR? without these changes, localization will work own container.

**Comment by YamatoAndo** [comment]
@mitsudome-r ping

File: common_sensor_launch/launch/velodyne_VLP16.launch.xml
Comment (mitsudome-r) [inline common_sensor_launch/launch/velodyne_VLP16.launch.xml]: It should be okay with capitalized "False", but it maybe better if it is aligned with other "true"/"false"

```suggestion
  <arg name="use_pointcloud_container" default="false"/>
```

File: common_sensor_launch/launch/velodyne_VLS128.launch.xml
Comment (mitsudome-r) [inline common_sensor_launch/launch/velodyne_VLS128.launch.xml]:
```suggestion
  <arg name="use_pointcloud_container" default="false"/>
```

File: common_sensor_launch/launch/velodyne_node_container.launch.py
Comment (mitsudome-r) [inline common_sensor_launch/launch/velodyne_node_container.launch.py]:
```suggestion
    add_launch_arg("use_pointcloud_container", "false")
```

**Review by mitsudome-r — APPROVED** [review]
LGTM.

I made a small suggestion, but I will leave it to you whether you want to make the change.

**Comment by kaancolak** [comment]
> LGTM.
> 
> I made a small suggestion, but I will leave it to you whether you want to make the change.

Thanks, Mitsudome-san. I applied your suggestion.

**Comment by kminoda** [comment]
@kaancolak @YamatoAndo Are there any other issues before merging this PR? (Sorry if I couldn't catch up related discussions)

**Comment by kaancolak** [comment]
@kminoda -san,  this [PR](https://github.com/autowarefoundation/autoware_launch/pull/108)  also relevant.  We can merge them together. 

The relevant issue is here : [1228](https://github.com/autowarefoundation/autoware.universe/issues/1228)

**Pull request merged**
